src/node/install.sh

```diff
@@ -285,23 +260,17 @@ elif [ "${NODE_VERSION}" = "latest" ]; then
     export NODE_VERSION="node"
 fi
 
-find_version_from_git_tags NVM_VERSION "https://github.com/nvm-sh/nvm"
-
 # Install snipppet that we will run as the user
 nvm_install_snippet="$(cat << EOF
-set -e
-umask 0002
-# Do not update profile - we'll do this manually
-export PROFILE=/dev/null
-curl -so- "https://raw.githubusercontent.com/nvm-sh/nvm/v${NVM_VERSION}/install.sh" | bash ||  {
-    PREV_NVM_VERSION=$(curl -s https://api.github.com/repos/nvm-sh/nvm/releases/latest | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
-    curl -so- "https://raw.githubusercontent.com/nvm-sh/nvm/\${PREV_NVM_VERSION}/install.sh" | bash
-    NVM_VERSION="\${PREV_NVM_VERSION}"
-}
-source "${NVM_DIR}/nvm.sh"
-if [ "${NODE_VERSION}" != "" ]; then
-    nvm alias default "${NODE_VERSION}"
-fi
+    set -e
+    umask 0002
+    # Do not update profile - we'll do this manually
+    export PROFILE=/dev/null
+    ./bash_install_node.sh
+    source "${NVM_DIR}/nvm.sh"
+    if [ "${NODE_VERSION}" != "" ]; then
+        nvm alias default "${NODE_VERSION}"
+    fi
```

Comment:
Can you remove the indentation here ? The original code seems correct

Reply:
sure